packages/preview-middleware-client/src/adp/quick-actions/common/add-controller-to-page.ts

```diff
@@ -0,0 +1,67 @@
+import OverlayRegistry from 'sap/ui/dt/OverlayRegistry';
+import FlexCommand from 'sap/ui/rta/command/FlexCommand';
+import UI5Element from 'sap/ui/core/Element';
+
+import { SIMPLE_QUICK_ACTION_KIND, SimpleQuickAction } from '@sap-ux-private/control-property-editor-common';
+
+import { getUi5Version } from '../../../utils/version';
+import { getAllSyncViewsIds, getControllerInfoForControl } from '../../utils';
+import { getRelevantControlFromActivePage } from '../../../cpe/quick-actions/utils';
```

Comment:
If `getRelevantControlFromActivePage` is needed in cpe and adp shouldn't this then be moved to `src/utils` or somewhere more central at src?